expo/expo-cli
# Added fig to introspection script

# Why

Auto generate scheme for fig terminal extension: https://github.com/withfig/autocomplete/blob/master/dev/expo.ts

- Added suggestions for `expo install`
- Added suggestions for `--max-workers`, `--config`, `--configuration`, `--scheme`, `--device` (iOS), `--platform`
- Added icons, descriptions, priority (sorted by help groups).
- Added help and version arguments.
- Strip ansii from strings.

# Test Plan

- fork `https://github.com/withfig/autocomplete`
  - Run `npm install` and `yarn dev`
- Run `yarn introspect fig` in `expo-cli/packages/expo-cli` (contents will be copied to clipboard)
- Paste generated script into `autocomplete/dev/expo.ts` and save
- Type `expo ` in the terminal to see the changes

- Test that `expo install ...` recommends packages

<img width="592" alt="Screen Shot 2021-07-28 at 5 25 10 PM" src="https://user-images.githubusercontent.com/9664363/127408801-87305aa0-3145-4a9b-851c-b0e1374585a6.png">

- Test that `expo run:ios --device ` replaces the device name with udid after selection



- Test that `expo run:ios --scheme` and `--configuration` recommend values (this one doesn't work reliably due to xcrun being trash).
- Test that `expo export --max-workers` recommends workers in best order for computer.
